src/test/utils.ts

```diff
@@ -1,5 +1,5 @@
-import schema from "schema"
-import { mergeSchemas } from "lib/mergeSchemas"
+// Please do not add schema imports here while stitching is an ENV flag
+//
```

Comment:
The move from static imports to dynamic is so that we can make guarantees about the merged schemas. It won't matter once it's removed from the env var though.